test/parallel/test-tls-env-extra-ca-with-ca.js

```diff
@@ -0,0 +1,55 @@
+// Certs in NODE_EXTRA_CA_CERTS are used for TLS peer validation
+
+'use strict';
```

Comment:
consistency nit, typical style is to have the `'use strict'` at the top

```suggestion
'use strict';

// Certs in NODE_EXTRA_CA_CERTS are used for TLS peer validation
```

Reply:
(likewise in the other test files)